Examples/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
We should probably remove the `Makefile` from all examples, no?

Reply:
I keep Makefile because they are good example in case you want to use Pcapp++ in project that use make.

But I'm not against removing them

Reply:
I'm not sure these `Makefile`s will even work with the new CMake build system 🤔